modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/operation/OperationMessageProcessor.java

```diff
@@ -191,7 +191,7 @@ public Event process(Event event) throws MuleException {
   }
 
   @Override
-  protected void doInitialise() throws InitialisationException {
+  public void doInitialise() throws InitialisationException {
```

Comment:
related to the prior comments, doInitialise() and doStart() should remain protected